src/controller/FrontPostController.php

```diff
@@ -13,56 +15,66 @@ class FrontPostController extends Constraints
 	/**
 	 * Displays the list of posts 
 	 *
-	 * @param $postModel
+	 * @param object $postModel
 	 * @param int $currentPage contains the page number
 	 *
 	 * @return array  contains post data 
 	 */
-	public function displayPosts($postModel, int $currentPage)
+	public function displayPosts(object $postModel, int $currentPage)
```

Comment:
```suggestion
	public function displayPosts(PostModel $postModel, int $currentPage)
```

Quand se sont des objets tu peux renseigner directement le nom de ta classe.

Reply:
OK, mais je dois laisser '@param object $postModel' sinon j'ai erreur sur '$countPosts' : 
$numberPosts = $countPosts->rowCount();

Expected type 'object'. Found 'array'